### PR TITLE
root: new version 6.30.04

### PR DESCRIPTION
**Pull request by wdconinc**
New version of ROOT, no changes to build system that aren't already accounted for (e.g. require http for webgui), https://github.com/root-project/root/compare/v6-30-02...v6-30-04.

### CODE REVIEW

**Review by tldahlgren — APPROVED** [review]
Confirmed the new version sha256.

**Comment by tldahlgren** [comment]
I would appreciate confirmation from at least one maintainer before merging.

**Review by HadrienG2 — APPROVED** [review]

**Review by vvolkl — APPROVED** [review]

**Pull request merged**
